cmd/start.go

```diff
@@ -496,6 +496,12 @@ func (x *Start) Execute(args []string) error {
 	proto.Unmarshal(dhtrec.GetValue(), e)
 
 	// Wallet
+	// Exchange rates
```

Comment:
I think this block should not be here. Exchange rates are already set in the `start`

Reply:
Ok fixed in the latest commit.